### PR TITLE
😮‍💨 Start revenue share modal

### PR DESCRIPTION
**Pull request by WRadoslaw**
#3845

### CODE REVIEW

**Comment by attemka** [comment]
@WRadoslaw 
1. enter any amount, delete it - NaN in both receive and spend.
2. 'You will spend' should be 'Your holders will receive'
3. Holders percentage is 0%
4. Transaction fee is missing(?)

**Comment by WRadoslaw** [comment]
> @WRadoslaw
> 
> 1. enter any amount, delete it - NaN in both receive and spend.
> 2. 'You will spend' should be 'Your holders will receive'
> 3. Holders percentage is 0%
> 4. Transaction fee is missing(?)

I will add fees during tx introduction to the flow

File: packages/atlas/src/components/_crt/StartRevenueShareModal/StartRevenueShareModal.tsx [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+      const now = base ? new Date(base.getTime()) : new Date()
+      now.setDate(now.getDate() + value.durationDays)
+      return now
+    }
+    return undefined
+  }, [])
+
+  return (
+    <DialogModal
+      title={`Sell $${title}`}
```
Comment (thesan) [inline packages/atlas/src/components/_crt/StartRevenueShareModal/StartRevenueShareModal.tsx]:
```suggestion
      title="Start revenue share"
```

File: packages/atlas/src/components/_crt/StartRevenueShareModal/StartRevenueShareModal.tsx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+      show={show}
+      onExitClick={onClose}
+      primaryButton={{
+        text: 'Sell tokens',
```
Comment (thesan) [inline packages/atlas/src/components/_crt/StartRevenueShareModal/StartRevenueShareModal.tsx]:
```suggestion
        text: 'Start revenue share',
```

**Review by thesan — COMMENTED** [review]
Just some copy changes

**Pull request merged**